openedx/core/djangoapps/oauth_dispatch/management/commands/tests/test_clear_expired_tokens.py

```diff
@@ -71,6 +71,11 @@ def test_excluded_application_ids(self):
                     'INFO',
                     f'Cleaned {0} rows from {RefreshToken.__name__} table'
                 ),
+                (
+                    LOGGER_NAME,
+                    'INFO',
+                    f'Cleaned {0} rows from {RefreshToken.__name__} table'
```

Comment:
Wondering if it is worth adding another argument to get a more precise log message, so it is more clear what is different here from the last message.

Reply:
That would require passing a few more arguments around and I'm trying to make as few changes as possible to be able to test this quickly